### PR TITLE
fix: Markdownlint cleanup for ZH-CN

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/zh-cn/web/api/node/nodename/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ var str = node.nodeName;
 | [Comment](/zh-CN/docs/DOM/Comment)                             | "#comment"                                                                                     |
 | [Document](/zh-CN/docs/DOM/document)                           | "#document"                                                                                    |
 | [DocumentFragment](/zh-CN/docs/DOM/DocumentFragment)           | "#document-fragment"                                                                           |
-| [DocumentType](/zh-CN/docs/DOM/DocumentType)                   | 等同于 [`DocumentType.name`](/zh-CN/docs/DOM/DocumentType.name)` 属性的值`                     |
+| [DocumentType](/zh-CN/docs/DOM/DocumentType)                   | 等同于 [`DocumentType.name`](/zh-CN/docs/DOM/DocumentType.name) 属性的值`                     |
```
Comment (yin1999) [inline files/zh-cn/web/api/node/nodename/index.md:29]:
```suggestion
| [DocumentType](/zh-CN/docs/DOM/DocumentType)                   | 等同于 [`DocumentType.name`](/zh-CN/docs/DOM/DocumentType.name) 属性的值                     |
```

File: files/zh-cn/web/api/webgl2renderingcontext/uniformmatrix/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: WebGL2RenderingContext.uniformMatrix[234]x[234]fv()
 slug: Web/API/WebGL2RenderingContext/uniformMatrix
 ---
-{{APIRef("WebGL")}} {{SeeCompatTable}}[WebGL 2 API ](/zh-CN/docs/Web/API/WebGL_API)的**`WebGL2RenderingContext.uniformMatrix[234]x[234]fv()`** 方法向 uniform 变量中传入指定的矩阵值。
+{{APIRef("WebGL")}} {{SeeCompatTable}}[WebGL 2 API](/zh-CN/docs/Web/API/WebGL_API) 的 **`WebGL2RenderingContext.uniformMatrix[234]x[234]fv()`** 方法向 uniform 变量中传入指定的矩阵值。
```
Comment (yin1999) [inline files/zh-cn/web/api/webgl2renderingcontext/uniformmatrix/index.md:5]:
```suggestion
{{APIRef("WebGL")}} {{SeeCompatTable}}

[WebGL 2 API](/zh-CN/docs/Web/API/WebGL_API) 的 **`WebGL2RenderingContext.uniformMatrix[234]x[234]fv()`** 方法向 uniform 变量中传入指定的矩阵值。
```

**Review by yin1999 — APPROVED** [review]
Thanks a lot. Some suggestions

File: files/zh-cn/learn/server-side/first_steps/web_frameworks/index.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ web 框架经常提供模板系统。这些允许你制定输出文档的结构
 
 框架提供一个机制，使得从存储的数据中生成其他格式数据变得简单，包括{{glossary("JSON")}}和{{glossary("XML")}}。
 
-比如，Django 模板允许你通过使用“双重花括号”（如` {`` { variable_name ``}``} `）来指定变量，当页面被渲染出来时，这些变量会被从视图函数传递过来的值代替。模板系统也会提供表达支持（通过语法`{% expression %}来实现`），这样就允许模板进行一些简单的操作比如迭代传递给模板的值列表。
+比如，Django 模板允许你通过使用“双重花括号”（如 `\{{ variable_name }}`）来指定变量，当页面被渲染出来时，这些变量会被从视图函数传递过来的值代替。模板系统也会提供表达支持（通过语法`{% expression %}来实现`），这样就允许模板进行一些简单的操作比如迭代传递给模板的值列表。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/first_steps/web_frameworks/index.md:142]:
```suggestion
比如，Django 模板允许你通过使用“双重花括号”（如 `\{{ variable_name }}`）来指定变量，当页面被渲染出来时，这些变量会被从视图函数传递过来的值代替。模板系统也会提供表达支持（通过语法 `{% expression %}` 来实现），这样就允许模板进行一些简单的操作比如迭代传递给模板的值列表。
```

**Comment by nschonni** [comment]
I've got files for one or two more of these once this one lands

**Comment by yin1999** [comment]
> I've got files for one or two more of these once this one lands

Thanks

**Pull request merged**